templates/telemetry/health/health.4.parquetmr.txt

```diff
@@ -0,0 +1,48 @@
+message health {
+  required binary type (UTF8);
+  required binary id (UTF8);
+  required binary creationDate (UTF8);
+  required double version;
+  required binary clientId (UTF8);
+  required group application {
```

Comment:
this should include the predefined template https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/templates/include/telemetry/application.1.parquetmr.txt